src/App.jsx

```diff
@@ -16,68 +17,85 @@ export const goods = [
 
 export const App = () => (
   <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+    <Goods />
+  </main>
+);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+function Goods() {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  function handleClick(good) {
+    setSelectedGood(good);
+  }
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  function removeClick() {
+    setSelectedGood('');
+  }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <>
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => removeClick()}
```

Comment:
```suggestion
            onClick={removeClick}
```
<img width="743" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/56c6222e-95a7-465e-91c5-de83e54e0708">